src/modules/layout/__tests__/HeaderDropDown.spec.js

```diff
@@ -0,0 +1,83 @@
+import React from 'react'
+import { shallow } from 'enzyme'
+import HeaderDropDown from '../components/HeaderDropDown'
+
+describe('HeaderDropDown', () => {
+  const MockNode = () => <div>Here comes the DropDown</div>
+  let wrapperComponent
+
+  beforeEach(() => {
+    wrapperComponent = shallow(<HeaderDropDown iconSrc='/someImg'>
+      <MockNode />
+    </HeaderDropDown>)
+  })
+
+  test('should be wrapped by onClickOutside', () => {
+    // todo: ??
+  })
+
+  test('should match snapshot', () => {
+    expect(wrapperComponent.dive()).toMatchSnapshot()
+  })
+
+  test('should pass correct closeDropDown callback', () => {
+    const component = wrapperComponent.dive()
+    expect(component.find('MockNode')).toHaveLength(1)
```

Comment:
You should be able to write `component.find(MockNode)` here. I'm not sure though. Would be interresting how the find() actually works :/

Reply:
I think I tried it and it didn't work

Reply:
k, already though that :P But I don't know why :/